docs/deploying-helium.md

```diff
@@ -319,36 +346,36 @@ $ az webapp create --resource-group {app_prefix}helium --plan {app_prefix}helium
 2. Set environment variables (which configure Helium):
 
 ```bash
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings COSMOSDB_URL="{Cosmos DB URL}"
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings KEY_VAULT_URL="{KeyVault URL}"
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings TENANT_ID={service principal tenand id}
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings CLIENT_SECRET={service principal password}
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings CLIENT_ID={service principal appId}
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings COSMOSDB_URL=$cosmosDBURL
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings KEY_VAULT_URL=$keyVaultURL
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings TENANT_ID=$heliumSPTenantId
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings CLIENT_SECRET=$heliumSPpw
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings CLIENT_ID=$heliumSPAppId
 ```
 
 Note: Database environment variables are automatically set to defaults for the MovieInfo reference app, unless otherwise specified:
 
 ```bash
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings DB_NAME={database name}
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings DB_COLLECTION={database collection}
-$ az webapp config appsettings set --resource-group {app_prefix}helium --name {prefix}helium --settings DEFAULT_PARTITION_KEY={default partition key}
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings DB_NAME={database name}
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings DB_COLLECTION={database collection}
+$ az webapp config appsettings set --resource-group $resourceGroupName -n $webAppName --settings DEFAULT_PARTITION_KEY={default partition key}
 ```
 
 3. Configuration via the Azure Portal is needed. Open the [Azure Portal](https://portal.azure.com) and perform the following actions:
-  - Go to your resource group _{app_prefix}helium_
-  - Go to your web application _{app_prefix}helium_
+  - Go to your resource group _$resourceGroupName_
```

Comment:
Can we do this from the CLI now? (I.e. no need to go to portal)

Reply:
I'm pretty sure we just used the CLI for the last hack in Bentonville.

Reply:
I couldn't find the config values to set in the CLI to replace this step in the Azure Portal -- let me know of any suggestion for where to find this.